src/sass/button.scss

```diff
@@ -8,10 +7,19 @@
     font-weight: 400;
     font-family: inherit;
     border-radius: $BORDER_RADIUS;
+    box-shadow: 0 2px 2px 0 $boxShadowColor;
+    transition-property: box-shadow, background;
+    transition-duration: 0.3s;
 
-    &:active, &:hover {
+    &, &:active, &:hover {
         text-decoration: none;
```

Comment:
can we add `outline: none` to this as well? to remove the blue outline when button is on focus

Reply:
Done. Thanks.